cli/input/input.go

```diff
@@ -48,16 +49,35 @@ func readLine(trm *term.Terminal, prompt string) (string, error) {
 
 // ReadPassword reads the user's password with prompt.
 func ReadPassword(prompt string) (string, error) {
-	trm := Terminal
-	if trm == nil {
-		s, err := term.MakeRaw(int(syscall.Stdin))
-		if err != nil {
-			return "", err
+	if runtime.GOOS == "windows" {
+		trm := Terminal
+		if trm == nil {
+			s, err := term.MakeRaw(int(syscall.Stdin))
+			if err != nil {
+				return "", err
+			}
+			defer func() { _ = term.Restore(int(syscall.Stdin), s) }()
+			trm = term.NewTerminal(ReadWriter{os.Stdin, os.Stdout}, prompt)
+
 		}
-		defer func() { _ = term.Restore(int(syscall.Stdin), s) }()
-		trm = term.NewTerminal(ReadWriter{os.Stdin, os.Stdout}, prompt)
+		return trm.ReadPassword(prompt)
+	}
+	f, err := os.OpenFile("/dev/tty", os.O_RDWR, 0)
```

Comment:
@roman-khimov, how can we test this code?

Reply:
You mean our regular tests? Create a pseudo-tty, for example.

Reply:
Could you, please, clarify, how can I create a pseudo-tty for this case?

Reply:
Something like https://pkg.go.dev/github.com/creack/pty ?

Reply:
Ugh, I'm stuck with it, I just can't make our tests work with it, so the current implementation is the best thing I can do for our tests. sshpass works with it as follows:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ sshpass -p q ./bin/neo-go contract deploy -i 1-print/1-print.nef -manifest 1-print/1-print.manifest.json -r http://seed1.neo.org:10332 -w wallet.json
Network fee: 0.00151452
System fee: 10.00104553
Total fee: 10.00256005
Relay transaction (y|N)> y
failed to push invocation tx: failed to send tx: InsufficientFunds (-500) - 
```